pages/directory/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Container, List, ListItem, Typography } from '@mui/material';
+import { NextPage } from 'next';
+import React from 'react';
+
+import createAxiosInstance from 'client/lib/axios';
+
+//TODO: Type users once sequelize-typescript is added
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Directory: NextPage<{ users: any[] }> = props => {
+  const { users } = props;
+
+  return (
+    <Container maxWidth="lg" className="pt-4">
+      <Box className="pt-4">
+        <Typography variant="h2" className="text-2xl">
+          Users:
+        </Typography>
+        <Box>
+          <List>
+            {users.map(user => (
+              <ListItem key={user.id}>{user.discord_name}</ListItem>
+            ))}
+          </List>
+        </Box>
+      </Box>
+    </Container>
+  );
+};
+
+Directory.getInitialProps = async ({ req }) => {
+  try {
+    const axios = createAxiosInstance(req);
+    const response = await axios.get('/api/users');
+    const users = response.data.users;
+
+    return { users };
+  } catch (error) {
+    return error;
```

Comment:
props should always be objects, but we should also aim to make them as consistent as possible.  might consider the following shape?

```ts
interface Props {
  users?: any[];
  error?: string;
}
```

then you can render an error if `users` is falsy or `error` is truthy